contracts/src/v0.8/l2ep/test/v1_0_0/scroll/ScrollValidator.t.sol

```diff
@@ -53,13 +53,19 @@ contract ScrollValidatorTest is L2EPTest {
   }
 }
 
-contract ScrollValidator_SetGasLimit is ScrollValidatorTest {
-  /// @notice it correctly updates the gas limit
-  function test_CorrectlyUpdatesTheGasLimit() public {
-    uint32 newGasLimit = 2000000;
-    assertEq(s_scrollValidator.getGasLimit(), INIT_GAS_LIMIT);
-    s_scrollValidator.setGasLimit(newGasLimit);
-    assertEq(s_scrollValidator.getGasLimit(), newGasLimit);
+contract ScrollValidator_Constructor is ScrollValidatorTest {
+  /// @notice it should have been deployed with the correct initial state
+  function test_InitialState() public {
```

Comment:
Please use the naming convention of 

`test_<function_being_tested>_<error_name(if any)_<success OR reverts>`

This is a bit harder with string errors as the name is very long. Maybe this ?

`test_constructor_InvalidL1MessageQueueAddress_reverts`?

Reply:
Fixed!